judo-runtime-core-jsl-itest/src/test/java/hu/blackbelt/judo/runtime/core/jsl/entity/AbstractTest.java

```diff
@@ -0,0 +1,538 @@
+                                jDao.create(J.builder().withNameI("V").build())
+                        ).stream().map(l -> JtoI(l)).collect(Collectors.toList()))
+                        .build(),
+                HAttachedRelationsForCreate.builder()
+                        .withRelationIOnHSingle(iOptionalRelation)
+                        .withRelationIOnHSingleRequired(iRequiredRelation)
+                        .withRelationIOnHMulti(iMultiRelation)
+                        .build()
+        );
+
+        //Check j and i instance ar equal because i is live with j only
+        assertEquals(iDao.getAll().size(), jDao.getAll().size());
+
+        //update
+
+        h.getCompositionIOnHSingle().get().setNameI("CompositionSingleChanged");
+        h.getCompositionIOnHSingleRequired().setNameI("CompositionSingleRequiredChanged");
+        h.getCompositionIOnHMulti().forEach(i -> i.setNameI("MultiCompositionNameChanged"));
+
+        iOptionalRelation.setNameI("iOptionalRelationNameChanged");
+        iRequiredRelation.setNameI("iRequiredRelationNameChanged");
+        iMultiRelation.forEach(i -> i.setNameI("iMultiRelationNameChanged"));
+
+
+        iDao.update(h.getCompositionIOnHSingle().get());
+        iDao.update(h.getCompositionIOnHSingleRequired());
+        h.getCompositionIOnHMulti().stream().map(i -> iDao.update(i));
+
+        iOptionalRelation = iDao.update(iOptionalRelation);
+        iRequiredRelation = iDao.update(iRequiredRelation);
+        iMultiRelation = iMultiRelation.stream().map(i -> iDao.update(i)).collect(Collectors.toList());
+
+        h = hDao.getById(h.identifier()).orElseThrow();
+
+        //Check update
+        assertEquals(Optional.of("CompositionSingleChanged"), h.getCompositionIOnHSingle().orElseThrow().getNameI());
+        assertEquals(Optional.of("CompositionSingleRequiredChanged"), h.getCompositionIOnHSingleRequired().getNameI());
+
+        //TODO JNG-4813
+        //h.getCompositionIOnHMulti().stream().forEach( i -> assertEquals(Optional.of("MultiCompositionNameChanged"),i.getNameI()));
+        //hDao.queryCompositionIOnHMulti.execute().stream().forEach( i -> assertEquals(Optional.of("iMultiRelationNameChanged"),i.getNameI()));
+
+        assertEquals(Optional.of("iOptionalRelationNameChanged"), hDao.queryRelationIOnHSingle(h).orElseThrow().getNameI());
+        assertEquals(Optional.of("iRequiredRelationNameChanged"), hDao.queryRelationIOnHSingleRequired(h).getNameI());
+        hDao.queryRelationIOnHMulti(h).execute().stream().forEach(i -> assertEquals(Optional.of("iMultiRelationNameChanged"), i.getNameI()));
+
+        //delete with IDao
+
+        iDao.delete(h.getCompositionIOnHSingle().get());
+        assertTrue(iDao.getById(h.getCompositionIOnHSingle().get().identifier()).isEmpty());
+        assertTrue(jDao.getById(h.getCompositionIOnHSingle().get().identifier().adaptTo(JIdentifier.class)).isEmpty());
+        assertEquals(iDao.getAll().size(),jDao.getAll().size());
```

Comment:
> iDao.getAll().size(),jDao.getAll().size()

Please don't forget Java styling conventions.

Also applies to other occurrences

Reply:
[6f85e57](https://github.com/BlackBeltTechnology/judo-runtime-core-jsl/pull/165/commits/6f85e57884996176f05ef4554ae4669dee9e50a2)